mercury.spec

```diff
@@ -1,8 +1,8 @@
-%global dl_version 2.0.0rc1
+%global dl_version 2.0.0rc2
```

Comment:
Just thinking out loud, but I wonder if we should be deriving this from `%{version}` since it's a known transform.

Not necessarily for this PR, but opening a new ticket for that would be useful I think.

Reply:
I filed this as https://jira.hpdd.intel.com/browse/DAOS-5838